src/cloudant/database.py

```diff
@@ -139,18 +140,36 @@ def delete(self):
 
     def all_docs(self, **kwargs):
         """
-
-        descending  Return the documents in descending by key order boolean false
-        endkey  Stop returning records when the specified key is reached string
-        include_docs    Include the full content of the documents in the return boolean false
-        inclusive_end   Include rows whose key equals the endkey  boolean true
-        key Return only documents that match the specified key  string
-        limit   Limit the number of the returned documents to the specified number  numeric
-        skip    Skip this number of records before starting to return the results  numeric 0
+        TODO: docstring worth a damn
+
+        descending  Return the documents in descending by key
```

Comment:
I think it should probably reference the API docs but need to work out a good link for that which wont change... definitely will param format before any kind of release tho